### PR TITLE
Fix for Content View entity in UI according to 6.2

### PR DESCRIPTION
**Pull request by oshtaier**
```
nosetests tests/foreman/ui/test_contentview.py
.S...SSS..S...EESSSS.ES.S....S.S.SESSSSSSS.SESSSSS.SSSSSSSSS..
======================================================================
----------------------------------------------------------------------
Ran 62 tests in 5421.327s

FAILED (SKIP=36, errors=5)

nosetests tests/foreman/ui/test_contentview.py -m test_positive_add_puppet_module
.
----------------------------------------------------------------------
Ran 1 test in 140.151s

OK

nosetests tests/foreman/ui/test_contentview.py -m rh_content
...
----------------------------------------------------------------------
Ran 3 tests in 367.850s

OK

nosetests tests/foreman/ui/test_contentview.py -m test_positive_create_composite
.
----------------------------------------------------------------------
Ran 1 test in 236.466s

OK
```

Bonus: quick fix for gpgkey bug decorators

### CODE REVIEW

**Comment by rochacbruno** [comment]
ACK

File: robottelo/ui/contentviews.py [hunk 1]
```diff
@@ -270,14 +270,7 @@ def add_puppet_module(self, cv_name, module_name, filter_term):
...
         """
-        element = self.search(cv_name)
-        if not element:
-            raise UIError(
-                'Could not find the selected CV "{0}"'.format(cv_name)
-            )
-
-        element.click()
-        self.wait_for_ajax()
+        self.search(cv_name).click()
```
Comment (abalakh) [inline robottelo/ui/contentviews.py]: `element.click()`?
Why not `self.click(self.search(cv_name))`?

File: robottelo/ui/contentviews.py [hunk 2]
```diff
@@ -477,16 +470,10 @@ def add_remove_errata_to_filter(self, cv_name, filter_name,
...
     def fetch_puppet_module(self, cv_name, module_name):
         """Get added puppet module name from selected content-view"""
-        element = self.search(cv_name)
-
-        if element is None:
-            raise UINoSuchElementError(
-                'Could not find the %s content view.' % cv_name)
-        element.click()
-        self.wait_for_ajax()
+        self.search(cv_name).click()
```
Comment (abalakh) [inline robottelo/ui/contentviews.py]: Same question

**Comment by abalakh** [comment]
ACK pending question

File: robottelo/ui/contentviews.py (hunk 1 above)
Comment (elyezer) [inline robottelo/ui/contentviews.py]: Good question.

**Comment by oshtaier** [comment]
Addressed and thanks for the comments
```
nosetests tests/foreman/ui/test_contentview.py -m test_positive_add_puppet_module
.
----------------------------------------------------------------------
Ran 1 test in 128.257s

OK
```

**Comment by elyezer** [comment]
ACK

**Pull request merged**